config/routes.rb

```diff
@@ -3,6 +3,8 @@
 
   root to: 'activities#index'
 
+  get 'projects', to: 'show_projects#index'
```

Comment:
Again, I'd vote for making this a `resources` route and eventually nesting it into the respective namespace.